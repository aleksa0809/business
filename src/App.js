import React, {Component} from 'react';
import { HashRouter, Route, Switch } from 'react-router-dom'
import Navbar from "./components/navbar"
import LeftMenu from "./components/leftmenu"
const Welcome   = React.lazy(() => import('./pages/welcome'));
const Instagram = React.lazy(() => import('./pages/instagram'));
const Contacts  = React.lazy(() => import('./pages/contacts'));
const Showcase  = React.lazy(() => import('./pages/showcase'));

class App extends Component {
    render() {
      return  (
        <div>  
            <Navbar />
            <div class="container">
                <div class="row">
                    <div class="col-lg-3">
                        <LeftMenu />
                    </div>
                    <div class="col-lg-9">
                      <HashRouter>
                        <Switch>
                          <Route path="/instagram">
                            <Instagram />
                          </Route>
                          <Route path="/contacts">
                            <Contacts />
                          </Route>
                          <Route path="/api/query/:sector/:size" render={props=> <Showcase {...props.match.params}/>} />
                          <Route path="/api/query/toplist" render={props=> <Showcase {...props.match.params} toplist="true"/>} />
                          <Route path="/api/query/basket"  render={props=> <Showcase {...props.match.params} basket="true" />}/>
                          <Route>
                            <Welcome />
                          </Route>
                        </Switch>
                      </HashRouter>
                    </div>
                </div>
            </div>
        </div>
        )
    }
}

export default App;
