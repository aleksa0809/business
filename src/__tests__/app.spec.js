import React from 'react';
import { render } from 'test-utils';
import App from '../app.jsx';

it('should take a snapshot', () => {
    const { asFragment } = render(<App />)
    
    expect(asFragment(<App />)).toMatchSnapshot()
});
