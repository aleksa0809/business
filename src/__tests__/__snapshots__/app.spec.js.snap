// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App--Container"
  >
    <p
      class="App--Title"
    >
      Welcome To React Boilerplate!
    </p>
    <img
      alt="React Logo"
      class="App--Image"
      src=""
    />
  </div>
</DocumentFragment>
`;
